### ISSUE TITLE
clair report as a single html 

### ISSUE THREAD

**Issue by superpoussin22** [SUSE/Portus#1369]
For the moment Portus is giving a liste of links pointing to the os CVE source.
whe must click also on each link to have a descitiption

it coudl be fine to have a report in html with a few details like the reg utility from Jess Frazzer is providing 

ling to jess reg tool and server 
https://github.com/jessfraz/reg/
https://github.com/jessfraz/reg/tree/master/server

hope it could be an enhancement for Portus

**Comment by mssola** [comment]
@superpoussin22 agreed, the UI is pretty raw right now... Thanks for the feedback 👍
